core/src/mill/define/BaseModule.scala

```diff
@@ -2,52 +2,53 @@ package mill.define
 
 import ammonite.ops.Path
 
-object BaseModule{
+object BaseModule {
   case class Implicit(value: BaseModule)
 }
 
 abstract class BaseModule(millSourcePath0: Path,
                           external0: Boolean = false,
-                          foreign0 : Boolean = false)
-                         (implicit millModuleEnclosing0: sourcecode.Enclosing,
-                          millModuleLine0: sourcecode.Line,
-                          millName0: sourcecode.Name,
-                          millFile0: sourcecode.File)
-  extends Module()(
-    mill.define.Ctx.make(
-      implicitly,
-      implicitly,
-      implicitly,
-      BasePath(millSourcePath0),
-      Segments(),
-      mill.util.Router.Overrides(0),
-      Ctx.External(external0),
-      Ctx.Foreign(foreign0),
-      millFile0
-    )
-  ){
+                          foreign0: Boolean = false)(
+    implicit millModuleEnclosing0: sourcecode.Enclosing,
+    millModuleLine0: sourcecode.Line,
+    millName0: sourcecode.Name,
+    millFile0: sourcecode.File)
```

Comment:
I really do not like this formatting =( it seems worse in every way compared to the original:

- Mixed indentation
- Mixed open/closing-paren alignment style

There are lots of other cases in this diff which are similarly changed. Could we configure scalafmt to maintain the old style?

Reply:
Dangling parens are now enabled by default (v1.6.0): https://github.com/scalameta/scalafmt/issues/1151  
Shouldn't there be a `.scalafmt.conf` in mill's root?

Reply:
There will be `.scalafmt.conf`, as a first attempt, I ran scalafmt on default configuration.

Reply:
@dwijnand I believe it was 1.5.1

Reply:
Hmm but didn't https://github.com/scalameta/scalafmt/issues/1151 ship in 1.6.0-RC1?